lib/timeline.js

```diff
@@ -31,7 +31,8 @@ Timeline.Pin = Pin;
  */
 Timeline.prototype.sendUserPin = function (userToken, pin, callback) {
   if (typeof userToken !== 'string') {
-    return callback(new Error('Expected userToken to be a string.'));
+    process.nextTick(callback(new Error('Expected userToken to be a string.')));
```

Comment:
This looks like it would call process.nextTick on the return value of the callback